app/src/main/java/com/safetyheads/akademiaandroida/data/ConfigRepositoryUseCaseImpl.kt

```diff
@@ -0,0 +1,45 @@
+package com.safetyheads.akademiaandroida.data
+
+import android.util.Log
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings
+import com.safetyheads.domain.entities.Config
+import com.safetyheads.domain.repositories.ConfigRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.tasks.await
+
+class ConfigRepositoryUseCaseImpl: ConfigRepository {
```

Comment:
Name isn't saying exactly what it should doing moreover you mixed with use case but it shouldn't. I suggest `FirebaseConfigRepostitory`